src/dvui.zig

```diff
@@ -7212,7 +7215,7 @@ pub fn pngFromTexture(arena: std.mem.Allocator, texture: Texture, opts: pngFromT
     @memcpy(ret[split + extra ..], png_bytes[split..@as(usize, @intCast(len))]);
 
     if (wasm) {
-        backend.dvui_c_free(png_bytes);
+        dvui.BackendWeb.dvui_c_free(png_bytes);
```

Comment:
not good but it works until we figure out a better way..

Reply:
Yeah I punted here